release-notes/10.0/preview/preview1/containers.md

```diff
@@ -0,0 +1,45 @@
+# Container image updates in .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 includes the following updates for container images:
+
+- [Ubuntu is now the default OS](#ubuntu-is-now-the-default-os)
```

Comment:
This arguable, but Ubuntu isn't really an OS. We haven't changed Linux as the default (actually only) OS for the convenience tags.

```suggestion
- [Ubuntu is now the default distro](#ubuntu-is-now-the-default-os)
```